tests/AppBundle/Mailjet/Message/AbstractEventMessageTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\AppBundle\Mailjet\Message;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Event;
+use AppBundle\Entity\PostAddress;
+
+abstract class AbstractEventMessageTest extends \PHPUnit_Framework_TestCase
+{
+    protected function createEventMock(string $name, string $beginAt, string $street, string $cityCode): Event
+    {
+        $address = PostAddress::createFrenchAddress($street, $cityCode)->getInlineFormattedAddress('fr_FR');
+
+        $event = $this->getMockBuilder(Event::class)->disableOriginalConstructor()->getMock();
```

Comment:
This new syntax is the shortcut for `$this->getMockBuilder(Event::class)->disableOriginalConstructor()->getMock();`